node/cn/gasprice/gasprice.go

```diff
@@ -156,19 +155,20 @@ func (gpo *Oracle) SuggestPrice(ctx context.Context) (*big.Int, error) {
 	*/
 }
 
-type getBlockPricesResult struct {
-	price *big.Int
-	err   error
-}
-
-type transactionsByGasPrice []*types.Transaction
-
-func (t transactionsByGasPrice) Len() int           { return len(t) }
-func (t transactionsByGasPrice) Swap(i, j int)      { t[i], t[j] = t[j], t[i] }
-func (t transactionsByGasPrice) Less(i, j int) bool { return t[i].GasPrice().Cmp(t[j].GasPrice()) < 0 }
-
-type bigIntArray []*big.Int
-
-func (s bigIntArray) Len() int           { return len(s) }
-func (s bigIntArray) Less(i, j int) bool { return s[i].Cmp(s[j]) < 0 }
-func (s bigIntArray) Swap(i, j int)      { s[i], s[j] = s[j], s[i] }
+// TODO-Klaytn-RemoveLater Later remove below obsolete code if we don't need them anymore.
```

Comment:
Commented out this codes since below codes were only used for old `SuggestPrice`, which are commented out above.